src/components/Map.tsx

```diff
@@ -20,9 +22,9 @@ type Props = {
   currentLocationEnabled?: boolean
   zoomLevel?: number
   mapCenter?: number[]
-  ownedHotspots?: Feature[]
-  selectedHotspots?: Feature[]
-  witnesses?: Feature[]
+  ownedHotspots?: Hotspot[]
```

Comment:
I changed the interface for these to take `Hotspots` so that the places where we're calling `Map` don't need to do any geo-logic